components/Buttons/src/MDCButton.m

```diff
@@ -338,11 +346,6 @@ - (void)touchesEnded:(NSSet *)touches withEvent:(UIEvent *)event {
 
   CGPoint location = [self locationFromTouches:touches];
   [_inkView startTouchEndedAnimationAtPoint:location completion:nil];
-
-  BOOL inside = CGRectContainsPoint(self.bounds, location);
-  if (inside && _shouldRaiseOnTouch) {
-    [self animateButtonToHeightForState:UIControlStateNormal];
```

Comment:
does this need to be called to initialize the button to the correct starting height? or will the proxying do it.